### PR TITLE
Add the pgfopts package

### PR DESCRIPTION
**Pull request by Kernald**

### CODE REVIEW

File: run_lualatex.py [hunk 1]
```diff
@@ -54,6 +54,7 @@
 env["TEXMFCNF"] = os.path.abspath("texmf/texmf-dist/web2c")
 env["TEXMFROOT"] = os.path.abspath("texmf")
 env["TTFONTS"] = ":".join(texinputs)
+env["max_print_line"] = "2147483647"
```
Comment (patbro) [inline run_lualatex.py]: Why is changing the `max_print_line` necessary for this package?

**Comment by Kernald** [comment]
Sorry it's not, this change comes from #62, I'll rebase that later today/tomorrow, it's not needed for this one.

File: run_lualatex.py (hunk 1 above)
Comment (Kernald) [reply run_lualatex.py]: Sorry it's not, this change comes from #62, I'll rebase that later today/tomorrow, it's not needed for this one.

**Comment by patbro** [comment]
I took a brief look at https://mirror.lyrahosting.com/CTAN/macros/latex/contrib/pgfopts/pgfopts.dtx but it doesn't seem easy to add a proper test. Or is it?

I would rather not merge this PR without a proper test. Then probably no test will be made at all.

**Comment by Kernald** [comment]
I don't know of an easy way to test this either. I'm not using it directly, I'm just trying to get it in as a transitive dependency of another package I'm looking to use.

**Comment by patbro** [comment]
Wouldn't it then be more logical to add `@texlive_texmf__texmf-dist__tex__latex__pgfopts` to your package locally using a patch instead?

**Comment by Kernald** [comment]
I think the situation is the exact same as #67 - adding a new package isn't exactly trivial, especially when dependencies include fonts etc. Ultimately, it's the same discussion we had in https://github.com/ProdriveTechnologies/bazel-latex/pull/58#issuecomment-894665747. I totally understand that you don't want to maintain additional packages that you're not using though - I think it would be worth having a clear decision written somewhere on whether you want to accept third party contributions of new packages or not. It's a bit confusing having a bunch of packages defined in this repo, and then some PRs closed because you're not using them.

(With that said, I'm well aware that this specific PR lacks tests, and I don't have an obvious solution to this.)

**Comment by patbro** [comment]
To be honest, making sure people can use bazel-latex in its full glory is more important than the additional work of reviewing all PRs and merging them. Until I have a proper solution to this, it's best to just merge those PRs. I will add a CONTRIBUTING.md file as well.

Hopefully, the number of packages doesn't become too big, which would cause the CI to take way too long.

**Pull request merged**